torchvision/datasets/sstem.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import print_function
+
+import os
+
+from PIL import Image
+import torch.utils.data as data
+
+from .utils import download_url, check_integrity
+from .vision import VisionDataset
+
+
+class ssTEM(VisionDataset):
+    """Dataset for `ISBI Challenge: Segmentation of neuronal structures
+    in EM stacks <http://brainiac2.mit.edu/isbi_challenge/>`_.
+
+    Args:
+        root (string): Root directory where dataset exists or will be saved
+            to if download is set to True.
+        train (bool, optional): If True, creates dataset from training set,
+            otherwise creates from test set.
+        transforms (callable, optional): A function/transform that takes in
+            two PIL images and returns transformed versions.
+        download (bool, optional): If true, downloads the dataset from the
+            internet and puts it in root directory. If dataset is already
+            downloaded, it is not downloaded again.
+    """
+    base_url = 'http://brainiac2.mit.edu/isbi_challenge/sites/default/files/'
+
+    meta = {
+        'train': {
+            'data': {
+                'filename': 'train-volume.tif',
+                'md5': '465461edbe0254630c4ec5577f1e7764'
+            },
+            'labels': {
+                'filename': 'train-labels.tif',
+                'md5': '657fe6b728c6dd0152e295c6d800001d'
+            }
+        },
+        'test': {
+            'data': {
+                'filename': 'test-volume.tif',
+                'md5': '9767660d7abe4e0ecbdd0061a16058ad'
+            }
+        }
+    }
+
+    def __init__(self, root, train=True, transforms=None, download=False):
+        # Lazy import
+        import skimage.io
+
+        super(ssTEM, self).__init__(root, transforms)
+
+        self.root = os.path.expanduser(root)
```

Comment:
With the [call of the superclass constructor](https://github.com/pytorch/vision/blob/50d54a82d1479ffb6dd7469ed05fccdf290a1d84/torchvision/datasets/vision.py#L10-L12), this is now obsolete.